pkg/kubelet/cm/event_dispatcher.go

```diff
@@ -41,8 +41,7 @@ type EventDispatcher interface {
 
 	// PostStopPod is invoked after all of a pod's containers have permanently
 	// stopped running, but before the pod sandbox is destroyed.
-	PostStopPod(pod *v1.Pod, cgroupPath string) (*lifecycle.EventReply, error)
-
+	PostStopPod(cgroupPath string) error
```

Comment:
with cpu isolation, pod struct is not needed in `PostStopPod` but if we want the interface to be very generic - maybe it'd be useful to have it sent to external isolator ? What do you think ?